test/runtests.bat

```diff
@@ -13,4 +13,4 @@ REM run the program, feed commands from input.txt file and redirect the output t
 java -classpath ..\bin seedu.addressbook.Main < input.txt > actual.txt
 
 REM compare the output to the expected output
-FC actual.txt expected.txt
\ No newline at end of file
+FC /W actual.txt expected.txt
```

Comment:
Actually there is no need to modify the runtest.bat or any scripts :)
If you found any errors in these files, feel free to post on the forum!